epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/LoggableHandler.java

```diff
@@ -99,6 +114,17 @@ public String logInvocation(Element element) {
     return builder.toString();
   }
 
+  /**
+   * Adds a log path "suffix" to the end of a single log invocation.
+   *
+   * @param logInvocation The log invocation generated by {@link LoggableHandler#logInvocation}
+   * @param suffix The suffix to add. This suffix is added "as-is" in code.
+   * @return the new log invocation
+   */
+  public String addLogPathSuffix(String logInvocation, String suffix) {
+    return logInvocation.replaceAll("\"\\)", "\" + " + suffix + ")");
```

Comment:
This seems overly broad. It'll replace any `")` token in the input, not necessarily only the ones you want

Reply:
Should I just search for backend.getNested("...") instead?